src/tt.h

```diff
@@ -63,6 +63,20 @@ struct TTEntry {
     int16_t  eval16;
 };
 
+struct TTData: public TTEntry {
+
+    TTData(TTEntry* tte) :
+        TTEntry(*tte),
+        ttEntry(tte){};
```

Comment:
I prefer `entry` as a member name over `ttEntry`. Also a semicolon after braces is redundant.